lib/core/src/client/preview/StoryRenderer.tsx

```diff
@@ -276,8 +276,10 @@ export class StoryRenderer {
   }
 
   renderDocs({ context, storyStore }: { context: RenderContext; storyStore: StoryStore }) {
-    const { kind, parameters } = context;
-
+    const { kind, parameters, id } = context;
```

Comment:
When storybook initially renders, the url contains id="*".

This is a non-valid storyId, but it's because the manager doesn't know a valid ID yet.

This code makes it so docs-mode doesn't throw when that happens.